versioned_docs/version-5/migration.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: 'Migrating from XState v4 to v5'
+---
+
+### Breaking changes
+
+`Machine` to `createMachine`
+
+`cond` is now `guard`
+
+Single-argument functions
+
+`(context, event)` => `({ context, event })`
+
+`actor.onTransition` is removed; use `actor.subscribe` instead
+
+`send` is deprecated; use `raise` or `sendTo` instead
+
+`internal: false` is now `reenter: true`
+
+`interpret(machine).start(state)` is now `interpret(machine, { state }).start()`
+
+`actor.send(eventObject)` no longer accepts a string; must pass full event object `{ type: '...' }`
+
+No more null events; use `always`
+
+2nd argument to `createMachine` is deprecated: `createMachine(config, options)` is now `createMachine(config).provide(options)`
+
+No more `.withConfig()`; use `.provide()` instead
+
+No more `.withContext()`; use input instead (see below)
```

Comment:
There's no below to see!

Reply:
WIP! 😅

Reply:
Of course! 😁

Would it be helpful to note things like that and also things that I notice are missing, or do you have it pretty well covered?

Reply:
Yes, that'd be helpful 🙏

Reply:
Oki will do!